saleor/giftcard/utils.py

```diff
@@ -49,3 +59,56 @@ def activate_gift_card(gift_card: GiftCard):
     if not gift_card.is_active:
         gift_card.is_active = True
         gift_card.save(update_fields=["is_active"])
+
+
+def gift_cards_create(
+    order: "Order",
+    gift_card_lines: Iterable["OrderLine"],
+    settings: "SiteSettings",
+    requestor_user: Optional["User"],
+    app: Optional["App"],
+    manager: "PluginsManager",
+):
+    """Create purchased gift cards."""
+    customer_user = order.user
+    user_email = order.user_email
+    gift_cards = []
+    non_shippable_gift_cards = []
+    for order_line in gift_card_lines:
+        price = order_line.total_price_gross
```

Comment:
We should handle situations when customers buy 5x gift card(5USD).
Current flow skips line quantity.